testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/cli/SuiteDescribe.java

```diff
@@ -232,7 +230,7 @@ public Execution(
             this.outputBuilder = describeOptions.format.outputBuilderFactory.get();
 
             try {
-                this.out = new PrintStream(new FileOutputStream(FileDescriptor.out), true, Charset.defaultCharset().name());
+                this.out = new PrintStream(System.out, true, Charset.defaultCharset().name());
```

Comment:
I think this is causing to actually write to stderr because of https://github.com/airlift/airlift/blob/master/log-manager/src/main/java/io/airlift/log/Logging.java#L139

This commit should be reverted. Logging to stderr here makes this fail to read any output: https://github.com/trinodb/trino/blob/master/.github/bin/build-pt-matrix-from-impacted-connectors.py#L108
and in result we're always running all PTs in PRs

Reply:
#17918